peerscout_api/main.py

```diff
@@ -99,6 +92,24 @@
     WHERE person.person_id IN UNNEST({person_ids})
 """
 
+LOGGER = logging.getLogger(__name__)
+
+
+class PersonProps(NamedTuple):
+    person_name: str
+    institution: Optional[str] = None
+    country: Optional[str] = None
+    availability: Optional[str] = None
+    Website_URL: Optional[str] = None
```

Comment:
very minor: it might be worth converting the url names to lower case for consistency (and general Python naming convention)